src/DES.java

```diff
@@ -0,0 +1,5 @@
+package src;
+
+public class DES extends AES {
```

Comment:
AES isn't a super class for DES.  This would be saying DES is a special type of AES when in fact they are two completely separate algorithms that both perform encryption.  Instead create a common parent class and have both DES and AES extend from that generic parent.